### ISSUE TITLE
IncompatibleClassChangeError - Exception on Android 4.4.4

### ISSUE THREAD

**Issue by almirvuk** [sthewissen/Xamarin.Forms.PancakeView#66]
PancakeView is only used for corner radius feature, but set_Elevation method is called and an exception is thrown because there is no support for Elevation on Android 4.4.4 version.

Packages:
- Xamarin.Forms Version: 4.2.0.709249
- Xamarin.Forms.PancakeView: 1.2.1

Tested device version:
- Device OS Android: 4.4.4

Stack trace:
```
Java.Lang.IncompatibleClassChangeError: no method with name='setElevation' signature='(F)V' in class Lcom/xamarin/forms/platform/android/FormsViewGroup;

JniEnvironment+InstanceMethods.GetMethodID (Java.Interop.JniObjectReference type, System.String name, System.String signature)
JniType.GetInstanceMethod (System.String name, System.String signature)
JniPeerMembers+JniInstanceMethods.GetMethodInfo (System.String encodedMember)
JniPeerMembers+JniInstanceMethods.InvokeVirtualVoidMethod (System.String encodedMember, Java.Interop.IJavaPeerable self, Java.Interop.JniArgumentValue* parameters)
View.set_Elevation (System.Single value)
PancakeViewRenderer.SetupShadow (Xamarin.Forms.PancakeView.PancakeView pancake)
```

**Comment by sthewissen** [comment]
Fixed in #65

**Comment by almirvuk** [comment]
Thank you for fixing this, and releasing package `1.3.6-beta`. The bug is fixed and working correctly on the tested device.
